prisma/schema.prisma

```diff
@@ -127,6 +127,65 @@ model Claim {
   lastUpdatedAt DateTime @default(now())
 }
 
+// this model is to connect between the claim and the data that supports it
+// like the name of the claim, the image, the source, etc.
+// maybe to add another atts in future like the urls that can be related to the claim, like the website, the social media, etc.
+model  ClaimData {
+  id      Int @id @default(autoincrement())
+  claimId Int @unique
+
+  name  String
+  // image Image[]
+}
```

Comment:
@Kholoudxs55kh i don't see how this ClaimData is used, now that the Image has the claimId in it; we do not need ClaimData i think?

We might want to just make validation claims with image source possibly, but anyway this is fine for now.